lib/grid.css

```diff
@@ -46,31 +42,46 @@
  */
 
 .Grid--alignRight {
-  text-align: right;
+  justify-content: flex-end;
 }
 
 /**
  * Modifier: middle-align grid cells
  */
 
-.Grid--alignMiddle > .Grid-cell {
-  vertical-align: middle;
+.Grid--alignMiddle {
+  align-items: center;
 }
 
 /**
  * Modifier: bottom-align grid cells
  */
 
-.Grid--alignBottom > .Grid-cell {
-  vertical-align: bottom;
+.Grid--alignBottom {
+  align-items: flex-end;
 }
 
 /**
- * Modifier: gutters
+ * Modifier: allow cells to equal distribute width
  *
- * NOTE: this can trigger a horizontal scrollbar if the component is as wide as
- * the viewport. Use padding on a container, or `overflow-x:hidden` to protect
- * against it.
+ * 1. Be explicit to work around IE10 bug with shorthand flex
+ *    https://github.com/philipwalton/flexbugs#6-the-default-flex-value-has-changed
+ */
+
+.Grid--fit > .Grid-cell {
+  flex: 1 1 0%; /* 1 */
```

Comment:
It's recommended not to because of IE10/11 - https://github.com/philipwalton/flexbugs#4-flex-shorthand-declarations-with-unitless-flex-basis-values-are-ignored